README.md

```diff
@@ -1,7 +1,71 @@
 [![pullreminders](https://pullreminders.com/badge.svg)](https://pullreminders.com?ref=badge)
 
 # zoo-event-stats
-Event stream and stats for the Zooniverse
+Event stream reader and JSON stats API for Zooniverse wide and project classification and comment counts.
+
+#### Stream Reader
+
+This service contains a Kinesis stream reader for the internal Zooniverse data stream.
+Each record on the input stream is processed if it's a talk comment or classification event.
+Processing invovles tranforming the incoming data and
+pushing data into both the Elastic Search data store and to the Pusher public data stream (public data only).
+
+See [KCLReader](./lib/input/kcl_reader.rb) and [start_stream](./bin/start_stream) for more details.
+
+#### API
+The stats service has a json API to query the event counts for projects
+
+##### `GET /counts/$event_type/$period`
+Get the counts of events that match the event type over the period.
+
+Valid `$event_type` values are `classification` and `comment`.
+
+Valid `$period` values are `minute`, `hour`, `day`, `week`, `month`, `quarter`, `year`.
+More details at [Value ES period values](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html#_calendar_intervals)
+
+This end points returns json data in the format below where the each period will a json object in the buckets list.
```

Comment:
~the~, might have forgotten a word between "will" and "a".

Reply:
thanks - feel free to use suggestions to fix these in the review :)

Reply:
```suggestion
This end points returns json data in the format below where the each period will return an object in the buckets list.
```